src/js/components/ResultsTable.jsx

```diff
@@ -5,11 +5,13 @@ import { debounce } from '../utils';
 
 export class ResultsTable extends PureComponent {
   componentDidMount() {
+    // set columns we want to be fixed width
+    const columnWidthData = { 1: 150 };
```

Comment:
Of course! Thanks for the review. I'll add them to the props on the <ResultsTable columnWidthData={...} />

I'll probably add it to `settings.js` as well, as this is something that will be set infrequently, but it should be configurable from a source when needed to be. This has worked well for me in other projects.